server/src/Irony.cpp

```diff
@@ -346,8 +356,16 @@ void Irony::complete(const std::string &file,
         }
       }
 
+      // Raise a warning if the candidate may be susceptible to:
+      // https://github.com/Sarcasm/irony-mode/issues/367
+      if (availability == CXAvailability_NotAccessible) {
+        brief = "Irony: Warning: Candidate: `" +
+                prototype +
+                "' maybe be inaccessible due to protected or private access."
+                "\n";
```

Comment:
This newline makes the minibuffer resizing even when not necessary (i.e. there is no brief).